examples/hello-world/hello_world.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Suite Setup                   Setup
+Suite Teardown                Teardown
+Test Setup                    Reset Emulation
+Test Teardown                 Test Teardown
+Resource                      ${RENODEKEYWORDS}
+
+*** Variables ***
+${UART}                       sysbus.uart0
+
+*** Keywords ***
+Create Machine
+    Execute Command          mach create
+    Execute Command          machine LoadPlatformDescription @platforms/boards/arduino_nano_33_ble.repl
+
+    Execute Command          sysbus LoadELF @${CURDIR}/binaries/hello_world/hello_world.ino.elf
+
+*** Test Cases ***
+Should Print Brightness Sequence
+    Create Machine
+    Create Terminal Tester    ${UART}
+
+    Execute Command           machine EnableProfiler "${CURDIR}/hello_world.dump"
```

Comment:
we should actually use the `@` notation, just like in LoadELF, whenever we provide a path. You can change that in other robot files as well, maybe just a PR to verify the tests and merge without a review